src/test/java/teammates/test/cases/webapi/EnrollStudentsActionTest.java

```diff
@@ -118,7 +118,7 @@ public void testExecute_withDuplicatedTeamNameAmongSectionsToExistingStudents_sh
         student1.section = "random section 1";
         StudentsEnrollRequest req = prepareRequest(Arrays.asList(student1));
         loginAsInstructor(typicalBundle.instructors.get("instructor1OfCourse1").getGoogleId());
-        verifyDuplicatedTeamNameDetected(courseId, req, student1.team, studentInCourse1.section, student1.section);
+        verifyDuplicatedTeamNameDetected(courseId, req, student1.team, student1.section, studentInCourse1.section);
```

Comment:
You need to add test cases at least for the two issues you solved

Reply:
Updated with new test cases